rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1115,7 +1115,9 @@ def __init__(
         self._create_metrics()
         self._update_metrics_to_log()
 
-        self.all_labels_embed = None  # needed for efficient prediction
+        # needed for efficient prediction
+        self.all_labels_embed: Optional[Tuple[tf.Tensor, tf.Tensor]] = None
```

Comment:
judging based on [this line](https://github.com/RasaHQ/rasa/blob/077d5dfa6140771113cd06dffd068d2c3d36e446/rasa/nlu/classifiers/diet_classifier.py#L1814) I'd say it should be 
```suggestion
        self.all_labels_embed: Optional[tf.Tensor] = None
```

Reply:
yes that's an issue I fixed in the other PR, don't know why I didn't fix it in this PR 🤷